Editor enhancements docs (#2222)
telerik/blazor-docs: _contentTemplates/editor/general.md

Blazor **Server** apps use the **SignalR WebSocket** to send the Editor `Value` from the browser to the server .NET runtime and vice-versa. The default SignalR maximum message size is **32 KB**. To work with larger content (especially paste images as Base64 strings), [increase the max WebSocket message size for the Blazor application]({%slug common-kb-increase-signalr-max-message-size%}).

#end
#prosemirror-schema-prerequisites
## Prerequisites

> Customization of the ProseMirror schema requires familiarity with:

Reviewer: Or perhaps an even simpler and general statement that can be applicable to all ProseMirror-related pages:

```suggestion
To work with ProseMirror, make sure you are familiar with:
```

> * JavaScript - the ProseMirror library is JavaScript-based and working with the schema is performed through JavaScript.

Reviewer: ```suggestion
* JavaScript&mdash;The ProseMirror library is JavaScript-based and the schema uses JavaScript syntax.
```

> * [ProseMirror Schema](https://prosemirror.net/docs/guide/#schema) - the schema structure and its children ([nodes](https://prosemirror.net/docs/ref/#model.NodeType), [marks](https://prosemirror.net/docs/ref/#model.MarkType)).

Reviewer: Nouns must be singular when used as adjectives: "child nodes".

#end

#prosemirror-schema-general-info

> The Editor accepts a custom ProseMirror schema through the `Schema` parameter(`string`). Use this parameter to provide the name of your JS function that returns your custom schema - an instance of the [ProseMirror `Schema` class](https://prosemirror.net/docs/ref/#model.Schema).

Reviewer: ```suggestion
The Editor accepts a custom ProseMirror schema through its `Schema` parameter. Set this parameter to the name of the JavaScript function that returns an instance of your custom [ProseMirror `Schema` class](https://prosemirror.net/docs/ref/#model.Schema).
```

> This JS function:

Reviewer: ```suggestion
This JavaScript function:

```

* Must be declared in the global scope (the `window` object).

> * Must return an instance of the [ProseMirror `Schema` class](https://prosemirror.net/docs/ref/#model.Schema)(the updated schema). You can access this class from the `ProseMirror` object of the event arguments.

Reviewer: This bullet repeats the sentence before the bullet list. Thus, you can further simplify the above sentence, e.g. "Set this `string` parameter to the name of the JavaScript function that:"

> * Provides an `arguments` object with the following properties:

Reviewer: "Provides" is unclear. A function either accepts (expects) arguments of specific type (and structure), or returns a value of specific type. Which one is it?

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Property | Description |
|----------|-------------|

> | `getSchema` | A function that returns the current [`Schema` object](https://prosemirror.net/docs/ref/#model.Schema). Before the Editor is initialized, the returned `Schema` object is the default schema of the Editor. After the Editor is initialized, the returned `Schema` object is the updated schema. Iy you don't provide a custom schema, this function always retrns the default schema. |

Reviewer 2: ```suggestion
| `getSchema` | A function that returns the current [`Schema` object](https://prosemirror.net/docs/ref/#model.Schema). Before the Editor is initialized, the returned `Schema` object is the default schema of the Editor. After the Editor is initialized, the returned `Schema` object is the updated schema. If you don't provide a custom schema, this function always returns the default schema. |
```

> | `getView` | A function that returns the currently used instance of [`EditorView` object](https://prosemirror.net/docs/ref/#view.EditorView). Before the Editor is initialized, the view (the result of the function) is null. |

Reviewer 2: ```suggestion
| `getView` | A function that returns the currently used instance of the [`EditorView` object](https://prosemirror.net/docs/ref/#view.EditorView). Before the Editor is initialized, the view (the result of the function) is null. |
```

| `ProseMirror` | An object that contains various ProseMirror classes and functions. |

> > You can set a custom schema only once—upon the initialization of the Editor component. Further changes to the schema will not be reflected. The Editor will continue to use the initially provided schema or, if no schema is initially provided, the built-in one.

Reviewer: ```suggestion
> You can set a custom schema only once during initialization of the Editor component. Further changes to the schema will not take effect and the component will continue using the initial custom or built-in schema.
```
